integration_tests/rollback/Rscripts/Rscripts.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
Should this file be .gitignored?

Reply:
nope - don't .gitignore Rproj since that way can easily launch in rstudio - which will properly give you the wd context etc.

Basically, between this + the makefile the entire test suite is set up to interactively use and poke very easily